plugins/flytekit-envd/tests/test_image_spec.py

```diff
@@ -25,6 +27,7 @@ def test_image_spec():
 
 def build():
     base(image="cr.flyte.org/flyteorg/flytekit:py3.8-latest", dev=False)
+    run(commands=["echo hello"])
```

Comment:
Curious: does this work in a real image build? If we have more than one command, does it have to use a list?

Reply:
The commands ["echo hello", "wget ..."] are the equivalent of
```Dockerfile
# Dockerfile
RUN echo hello
RUN wget ...
```
Yes, you can definitely use `&&` as well. like commands["echo hello && wget ..."]